src/pages/compendia/[type]/download/[organism_name].js

```diff
@@ -0,0 +1,126 @@
+  const { setResponsive } = useResponsive()
+  const { error, downloadUrl } = useDownloadCompendium(compendium)
+
+  if (error) {
+    return (
+      <FixedContainer>
+        <Error
+          statusCode={error}
+          align="center"
+          direction="column"
+          marginTop="none"
+        />
+      </FixedContainer>
+    )
+  }
+
+  // shows Spinner until the download URL is ready
+  if (!downloadUrl) return <Spinner />
+
+  return (
+    <FixedContainer>
+      <Box
+        align="center"
+        pad={{
+          top: 'large',
+          bottom: 'xlarge'
+        }}
+      >
+        <Row justify="center" width={setResponsive('100%', '70%')}>
+          <Column align={setResponsive('center', 'start')}>
+            <Box direction="row" gap="xxsmall" margin={{ bottom: 'small' }}>
+              <Heading level={1}>
+                <Icon color="success" name="Success" /> Downloading{' '}
+                {formatString(compendium.primary_organism_name || '')}{' '}
+                compendium...
+              </Heading>
+            </Box>
+            <Box direction="start" gap="xsmall">
+              <Paragraph>If the download did not start,</Paragraph>
+              {downloadUrl && (
+                <Button
+                  label="click here."
+                  href={downloadUrl}
+                  link
+                  linkFontSize="16px"
+                />
+              )}
+            </Box>
+          </Column>
+          <Column
+            align="center"
+            margin={{
+              top: setResponsive('large', 'none'),
+              bottom: setResponsive('large', 'none'),
+              left: setResponsive('none', 'medium', 'basex13')
+            }}
+          >
+            <Box
+              aria-hidden
+              background={{
+                image: "url('/illustration-dataset.svg')",
+                position: 'center',
+                repeat: 'no-repeat',
+                size: 'contain'
+              }}
+              // to preserve the dimension of SVG image
+              height={setResponsive('169px', '169px', '169px')}
+              width={setResponsive('250px', '210px')}
+            />
+          </Column>
+        </Row>
+      </Box>
+      <Box>
+        <Explore />
+      </Box>
+    </FixedContainer>
+  )
+}
+
+export const getServerSideProps = async ({ query }) => {
+  const { type, organism_name: organismName } = query
+  // The routes must be the valid compendia types
+  if (!compendia.types.includes(type)) {
+    return {
+      notFound: true
+    }
+  }
+
+  const compendiaQuery = {
+    latest_version: true,
+    limit: 1000,
+    quant_sf_only: type === 'rna-seq'
+  }
+
+  const response = await api.compendia.get(compendiaQuery)
+
+  // finds the compendium that matches organismName
+  const compendium = response.results.find(
+    (organism) => organism.primary_organism_name === organismName
+  )
+
+  return {
+    props: {
+      compendium: compendium || {},
```

Comment:
Is this necessary? Would passing a null value matter in this instance if we also pass `notFound`? Similar to the other compendia page?